### PR TITLE
bug/EUI-2974-link-issue-as-it-get-undefined-keyvalue-before-subscription

### PR DESCRIPTION
**Pull request by rayliangatos**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by udaydenduluri33 — APPROVED** [review]
LGTMr

**Review by Vamshi-Muniganti — APPROVED** [review]
looks good to me

File: charts/xui-mo-webapp/values.preview.template.yaml [hunk 1]
```diff
@@ -14,8 +14,8 @@ nodejs:
     #SERVICES_TERMS_AND_CONDITIONS_API_SERVICE: http://xui-terms-and-conditions-aat.service.core-compute-aat.internal
     #S2S_SERVICE: http://rpe-service-auth-provider-aat.service.core-compute-aat.internal
     #FEE_AND_PAY_API: https://payment-api-aat.service.core-compute-aat.internal
-    MANAGE_CASE_LINK: https://xui-webapp-aat.service.core-compute-aat.internal/cases
-    MANAGE_ORG_LINK: https://xui-mo-webapp-aat.service.core-compute-aat.internal/
+    MANAGE_CASE_LINK: https://manage-case.aat.platform.hmcts.net/
```
Comment (ernestman28) [inline charts/xui-mo-webapp/values.preview.template.yaml:17]: should this not be?

MANAGE_CASE_LINK: https://manage-case.aat.platform.hmcts.net/cases

File: charts/xui-mo-webapp/values.aat.template.yaml [hunk 2]
```diff
@@ -16,8 +16,8 @@ nodejs:
     #S2S_SERVICE: http://rpe-service-auth-provider-aat.service.core-compute-aat.internal
     APPINSIGHTS_INSTRUMENTATIONKEY: ${APPINSIGHTS_INSTRUMENTATIONKEY}
     #FEE_AND_PAY_API: https://payment-api-aat.service.core-compute-aat.internal
-    MANAGE_CASE_LINK: https://xui-webapp-aat.service.core-compute-aat.internal/cases
-    MANAGE_ORG_LINK: https://xui-mo-webapp-aat.service.core-compute-aat.internal/
-    
+    MANAGE_CASE_LINK: https://manage-case.aat.platform.hmcts.net/
```
Comment (ernestman28) [inline charts/xui-mo-webapp/values.aat.template.yaml:19]: same?

MANAGE_CASE_LINK: https://manage-case.aat.platform.hmcts.net/cases

**Review by ernestman28 — APPROVED** [review]

**Pull request closed without merging**